### ISSUE TITLE
Fix template and css

### ISSUE THREAD

**Issue by maclandrol** [legaultmarc/genomenews#6]
Fix all template style (form and details)
Fix windows resizing
Add client side functionality (vote effect, etc ) with jquery

**Comment by legaultmarc** [comment]
I disagree with the effects. I think we should stick to something simple like news.ycombinator and datatau. If we really want to have a "fancy" version, perhaps we could try to implement it as another optional template or stylesheet.

**Comment by maclandrol** [comment]
Not a fancy stuff. Just a feedback when user click on upvote like change the arrow color to indicate that the vote was saved.
HN and datatau make the upvote button disappears which in my opinion is not the best way to do it. 
I prefer a version where I could "undo my upvote"  if I change my mind (accidentally upvoted the wrong post for example)

And for the layout. The resizing is actually not working

![image](https://cloud.githubusercontent.com/assets/5290110/5785778/fb1d036c-9da8-11e4-8c58-ce767244c619.png)

![image](https://cloud.githubusercontent.com/assets/5290110/5785779/ff656a72-9da8-11e4-8a36-a31356b5346f.png)
